examples/twri/scripts/GaussLens_inversion.jl

```diff
@@ -0,0 +1,92 @@
+################################################################################
+#
+# Run inversion for the Gaussian lens model
+#
+################################################################################
+
+R = Float32
+
+### Module loading
+using LinearAlgebra, JLD2, PyPlot, Dates, Distributed
+using Optim, LineSearches
```

Comment:
Are `Optim` and `LineSearches` in the dependencies or described in `twri/examples/README.md`?

Reply:
Added to the README in `examples/README.md`